spec/system/posts_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe 'ポスト機能', type: :system do
+  describe 'ポストの表示機能' do
+    let!(:post) { create(:post, contents: 'hogehoge') }
+
+    it 'ポスト一覧画面で表示されること' do
```

Comment:
root_pathだから

```suggestion
    it 'トップページで表示されること' do
```

かな？

Reply:
1c29470792d96d1c90bba7d709340c8e5f812c41

修正しました！